tests/test_modbus_block.py

```diff
@@ -94,7 +100,7 @@ def test_exception_code(self, mock_client):
                          'are not allowed or do not exist in slave')
         blk.stop()
 
-    @patch('modbus.modbus_block.sleep')
```

Comment:
@mattdodge, was this failing or is this just a way to make the test still work if the module moves?